sdk/translation/Azure.AI.Translation.Document/api/Azure.AI.Translation.Document.netstandard2.0.cs

```diff
@@ -120,6 +120,8 @@ public partial class DocumentTranslationOperation : Azure.PageableOperation<Azur
         public override Azure.AsyncPageable<Azure.AI.Translation.Document.DocumentStatus> GetValuesAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override Azure.Response UpdateStatus(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override System.Threading.Tasks.ValueTask<Azure.Response> UpdateStatusAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public override Azure.Response<Azure.AsyncPageable<Azure.AI.Translation.Document.DocumentStatus>> WaitForCompletion(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I am a little torn about exposing this or not, and the more I look at it, I think I should leave it as internal...
So the dilemma I am having is that this is a sync method, but the return type is an `AsyncPageable` because `PageableOperation` by default is `PageableOperation<AsyncPageable<T>>`. 

A user could do:
```
var operation = client.Translation(input);
foreach (DocumentStatus document in operation.GetValues())
```
and they won't notice a thing and in reality, everything is sync.

But if they do:
```
var operation = client.Translation(input, waitforCompletion: false);
var results = operation.WaitForCompletion();
async foreach (DocumentStatus document in results) // this will be very weird :|
```

What do you think

Reply:
I don't have any great ideas here, unfortunately. Let's raise it to the team.

Reply:
I am hiding it for now from the user